pkgs/by-name/po/poptracker/package.nix

```diff
@@ -53,10 +57,28 @@ stdenv.mkDerivation (finalAttrs: {
     runHook preInstall
     install -m555 -Dt $out/bin build/linux-x86_64/poptracker
     install -m444 -Dt $out/share/${finalAttrs.pname} assets/*
-    wrapProgram $out/bin/poptracker --prefix PATH : ${lib.makeBinPath [ which libsForQt5.kdialog ]}
+    wrapProgram $out/bin/poptracker --prefix PATH : ${
+      lib.makeBinPath [
+        which
+        libsForQt5.kdialog
+      ]
+    }
     runHook postInstall
   '';
 
+  desktopItems = [
+    (makeDesktopItem {
+      name = "poptracker";
+      desktopName = "PopTracker";
+      exec = "poptracker";
+      comment = "Universal, scriptable randomizer tracking solution";
+      categories = [
+        "Game"
+        "Utility"
```

Comment:
any icon we can use?

Reply:
Is there a way to reference the `$out` directory from the makeDesktopItem call? Poptracker does include an icon in its assets directory(installed as `$out/share/poptracker`), but I couldn't find a way to reference it from the makeDesktopItem call.

Reply:
If it's installed into the correct dir (`$out/share/pixmaps/` or `$out/share/icons/hicolor/<size>/apps/`), simply the name of the icon should suffice.

Reply:
Added an icon